bin/run-single-tv-datasetcreation-pipeline.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Fuck in my head TV expands to transcription factor. I'll fix it

On Aug 1, 2016 9:38 AM, "akmorrow13" <notifications@github.com> wrote:

> In bin/run-single-tv-datasetcreation-pipeline.sh
> <https://github.com/akmorrow13/endive/pull/1#discussion_r73010165>:
>
> > @@ -0,0 +1,8 @@
> > +#!/bin/bash
>
> whats a tv?
> run-single-tv-datasetcreation-pipeline.sh
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/akmorrow13/endive/pull/1/files/b00dbd128239b200b96c55e928cc42abdd9b4805#r73010165>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABTEkEbqX0LnyGtmJUx-_DSDXCfe9rKQks5qbiD3gaJpZM4JZw3M>
> .
>